cirq/study/resolver.py

```diff
@@ -80,6 +80,9 @@ def value_of(self,
                 return float(v)
         return value
 
+    def __iter__(self):
+        return self.param_dict.__iter__()
+
```

Comment:
What's this for?

Reply:
I have to iterate across the keys of param resolvers in order to transform them into zips in the proto.  Basically, I have to transpose {'a':1,'b':2},{'a':3,'b':4} into a zip of 'a' {1,3} zipped with b {2,4}.  In order to do that, I need to iterate across the keys of the param resolver.